File: Ko.PoC.ViewModels/HelloWorldViewModel.cs
using Ko.Mvc.KnockoutAttributes;

namespace Ko.PoC.ViewModels
{
    public class HelloWorldViewModel
    {
        public string FirstName { get; set; }
        public string LastName { get; set; }

        [KnockoutComputed]
        public string FullName { get { return FirstName + " " + LastName; } }
    }
}

File: Ko.PoC.Web/Scripts/ViewModels/HelloWorldViewModel.js
Ko.PoC.ViewModels.HelloWorldViewModel.prototype.InitExtensions = function () {
    var self = this;    

    /*------------------computed properties------------------*/    
    self.FullName = ko.computed(function(){
        //TODO: Write computing logic here:
        return self.FirstName() + " " + self.LastName();
    });
    /*-------------------------------------------------------*/
    return self;
};
